packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -46,6 +46,27 @@
     }
   }
 
+  .toggler-border {
```

Comment:
generally try not to name css from what it looks like but rather from what it is. E.g. toggler-wrapper or something